### DISCUSSION TITLE
labelOpts 里的onClick事件不生效

### DISCUSSION THREAD

**Discussion by BerryLe** [zouyaoji/vue-cesium#462] [Q&A]
pinDrawingOpts: {
          billboardOpts: {
            image: 'https://zouyaoji.top/vue-cesium/images/grepin.png',
            onClick(e) {
              console.log(e)
            }
          },
          labelOpts: {
            text: '图标点',
            pixelOffset: [0, -60],
            onClick(e) {
              console.log(e)
            }
          }
        },
        
        这个labelOpts 里的onclick事件不生效

**Comment by zouyaoji** [comment]
为了让label处于所有对象的最上面，被特殊处理过，导致无法被scene.pick拾取到，如果要被拾取到，参考图示加个参数 `disableDepthTest` 即可 
![image](https://github.com/zouyaoji/vue-cesium/assets/24430116/cb00cba3-9882-4343-98b2-dc2f4d1bdaca)

但这样文字可能被其他对象遮挡，此时还可以尝试加大文字的深度检测距离参数 `labelOpts.disableDepthTestDistance`
